shanoir-ng-datasets/src/main/java/org/shanoir/ng/download/WADODownloaderService.java

```diff
@@ -139,6 +149,17 @@ public void downloadDicomFilesForURLs(final List<URL> urls, final File workFolde
 			}
 		}
 	}
+	
+	public String downloadDicomMetadataForURL(final URL url) throws IOException, MessagingException {
+		if (url != null) {
+			String urlStr = url.toString();
+			if (urlStr.contains(WADO_REQUEST_STUDY_WADO_URI)) urlStr = wadoURItoWadoRS(urlStr);
```

Comment:
This is based on the supposition that the PACS accepts REST requests, but it's not always the case no? Am I wrong @michaelkain ?
Maybe you should test
if (dicomWeb) { REST } else { WadoURI}

Reply:
The base for this PR is a new PACS. Without it neither OHIF viewer, nor re-apply studycards nor quality control will work. We have not yet changed to store paths in wado-rs, so it should be fine.